<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNNControl</class>
 <widget class="QMainWindow" name="CNNControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CNNControl</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Network Options</string>
    </property>
    <widget class="QPushButton" name="but_OpenCNN">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Network</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_SaveCNN">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>112</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Network</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Genome Options</string>
    </property>
    <widget class="QPushButton" name="but_LoadGenome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Genome</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_LoadGenome_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Genome</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_LoadGenome_TW">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Genome TW</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
